### ISSUE TITLE
Could not load any of ['hdf5.dll', 'hdf5dll.dll', 'pytables_hdf5.dll'], please ensure that it can be found in the sys tem path.

### ISSUE THREAD

**Issue by WestXu** [PyTables/PyTables#779]
After install tables 3.6.1, this happens: 
```
PS D:\git-project> ipython
Python 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.9.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import tables
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-0a99ad28a088> in <module>
----> 1 import tables

d:\apps\python37\lib\site-packages\tables\__init__.py in <module>
     81             dll_loc = 'is installed in the package folder'
     82         raise ImportError(
---> 83             'Could not load any of %s, please ensure that it %s.' % (hdf5_dlls, dll_loc))
     84
     85     # Some PyTables binary distributions place the dependency DLLs in the

ImportError: Could not load any of ['hdf5.dll', 'hdf5dll.dll', 'pytables_hdf5.dll'], please ensure that it can be found in the sys
tem path.
```

`pip install tables==3.5.2` fixes it.

**Comment by tomkooij** [comment]
:-(

I'll have a look.

**Comment by tomkooij** [comment]
Sadly, I can reproduce this... All windows wheels seem to be affected. 

The strange thing is that `hdf5.dll` is both packaged and installed with the wheel. *and* the wheel is tested (in a separate env) and works on the wheelbuilder.

I'll have a look at the wheel builder, to try to figure out what is wrong.

**Comment by tomkooij** [comment]
I don't really understand why the wheels are broken. And I won't invest any time in the current windows wheelbuilder.

I've got an experimental windows wheelbuilder (that vendores `hdf5.dll` as `pytables_hdf5.dll` to prevent DLL Hell) that I need to (and will) finish.

I've replaced the py38, py37 and py36 windows wheels with my experimental builds. These at least seems to work, but probably need some fine tuning.

Sadly, the py35 builders isn't working yet... So the python 3.5 wheels are still broken for now.

**Comment by cgohlke** [comment]
The `hdf5.dll` in the wheels on PyPI depends on `zlib.dll`, which is not distributed.

**Comment by tomkooij** [comment]
@cgohlke : Thanks for helping out! However: `hdf5.dll` (from the wheelbuilder, build by conda) does indeed depend on `zlib.dll`: This applies to versie `3.6.1` of the wheels.

The wheels currently on PyPI (version `3.6.1-2`) are built against `pytables_hdf5.dll` which is statically linked against zlib, and does not require `zlib.dll`. That's why it is not included in the wheel.

(I checked with dependency walker, but please let me know if I'm wrong)

**Comment by cgohlke** [comment]
Yes, you are right. I was referring to `3.6.1` wheels which contain `hdf5.dll`, not `3.6.1-2`.

**Comment by raphaelquast** [comment]
I've been running into similar issues using `conda install pytables`   (resulting in pytables 3.6.1)
(on a pc running Windows Server 2012 RC2)
```python
import tables
>>>Traceback (most recent call last):
>>>
>>>  File "<ipython-input-353-0a99ad28a088>", line 1, in <module>
>>>    import tables
>>>
>>>  File "... path to conda-env ... \lib\site-packages\tables\__init__.py", line 99, in >>><module>
>>>    from .utilsextension import (
>>>
>>>ImportError: DLL load failed: The specified module could not be found.
```

however, uninstalling it and **re-installing via pip**, e.g. `pip install tables` solves the problem!
(not sure what's happening since it's still v3.6.1)

**Comment by tomkooij** [comment]
Might be related: https://ci.appveyor.com/project/PyTablesBot/pytables/builds/29773046/job/xed63kk50g76y5lv

**Comment by tomkooij** [comment]
Fixed in #787
